forem/forem
# Add new ColorPicker to organization settings

## What type of PR is this? (check all applicable)


## Description

This PR adds the new ColorPicker component to the Organization settings page (this includes both editing an existing organization, and creating a new one).

This was also the last place that used the old `colorPreview` pack file, so this has now been removed too.

## Related Tickets & Documents



- Related Issue https://github.com/forem/forem/issues/15179

## QA Instructions, Screenshots, Recordings

- Login and visit `/settings/organization`
- Create a new organization by choosing "New organization" from the dropdown
- Check that the Brand color pickers (1 and 2) work as expected
- Fill out the other required fields and save the organization
- Try editing your new organization, changing the colors and making sure the changes are saved


https://user-images.githubusercontent.com/20773163/154697276-0e5d1f29-ff7d-4e2a-a9fd-423c22901139.mp4

### UI accessibility concerns?

This provides an accessibility boost for the color pickers in this form

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: once all pickers are replaced, the overarching project will be communicated


## [optional] What gif best describes this PR or how it makes you feel?

![Benedict Cumberbatch with rainbow hands](https://media.giphy.com/media/fbXyEHrV6n2Kc/giphy.gif)
